ArduPlane/quadplane.cpp

```diff
@@ -3583,4 +3598,13 @@ bool QuadPlane::show_vtol_view() const
     return show_vtol;
 }
 
+// return the tailsitter.transition_angle_vtol value if non zero, otherwise returns the tailsitter.transition_angle_fw value.
+AP_Int8 QuadPlane::get_tailsitter_transition_angle_vtol() const
+{
+    if (tailsitter.transition_angle_vtol == 0) {
+        return tailsitter.transition_angle_fw;
```

Comment:
```suggestion
        return tailsitter.transition_angle_fw.get();
```

Reply:
why all the gets?

Reply:
the gets give you a `Int8_t` rater than a `AP_Int8`

Reply:
have been told that this should be used only if compiler demands since it adds unnecessarily to execution path...we can discuss at DEVCALL since I dont know which is correct...

Reply:
I'll let you off the `get()` but the function should be returning a normal `Int8_t`

Reply:
fixed to int8_t